### PR TITLE
Update kube component reference docs

### PR DESCRIPTION
**Pull request by tengqm**
The upstream repo and tools are now including deprecated flags (e.g.
`--admission-controls` back in the help message output
(i.e. `kube-apiserver --help`). This PR changes the doc generation logic
accordingly to avoid confusion.

Closes: #9367

### CODE REVIEW

**Comment by steveperry-53** [comment]
@tengqm I don't see where you changed doc generation logic. Did you change a generator, and use the generated output to populate this PR? Or are the changes in this PR manual and temporary?

**Comment by tengqm** [comment]
@steveperry-53 It is now in master: https://github.com/kubernetes-incubator/reference-docs/pull/52.

File: content/en/docs/reference/command-line-tools-reference/cloud-controller-manager.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ notitle: true
 ## cloud-controller-manager
 
 
+
```
Comment (stewart-yu) [inline content/en/docs/reference/command-line-tools-reference/cloud-controller-manager.md]: Delete it

**Comment by stewart-yu** [comment]
If i'm not misunderstand, component's parameter are auto-generated

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md [hunk 2]
```diff
@@ -20,94 +20,80 @@ kubeadm upgrade apply [version]
...
-  </colgroup>
-  <tbody>
-
     <tr>
-      <td colspan="2">--allow-experimental-upgrades</td>
+      <td colspan="2">--feature-gates string</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">Show unstable versions of Kubernetes as an upgrade alternative and allow upgrading to an alpha/beta/release candidate versions of Kubernetes.</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">A set of key=value pairs that describe feature gates for various features.Options are:<br/>Auditing=true|false (ALPHA - default=false)<br/>CoreDNS=true|false (default=true)<br/>DynamicKubeletConfig=true|false (ALPHA - default=false)<br/>SelfHosting=true|false (ALPHA - default=false)<br/>StoreCertsInSecrets=true|false (ALPHA - default=false)</td>
```
Comment (stewart-yu) [inline content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md]: DynamicKubeletConfig, is Beta, default value if true now

**Comment by zacharysarah** [comment]
@tengqm 👋 Any updates on review feedback?

**Comment by zacharysarah** [comment]
@tengqm 👋 Please feel free to reopen this PR when it's ready for merge.

**Pull request closed without merging**